repository_rules.bzl

```diff
@@ -6,9 +6,55 @@ This is adapted from
 https://github.com/googleapis/googleapis/blob/master/repository_rules.bzl
 """
 
-load("//:remote_apis_deps.bzl", "remote_apis_go_deps")
 
 def _switched_rules_impl(ctx):
+    go_deps = """
```

Comment:
Is there a way to make this an actual bzl file that we include on demand, rather than a string inside this one? That way, it formats/lints nicely in various IDEs.

Reply:
I concur with Ola: Ideally this would be a standalone bzl file, rather than something generated from a string.

Reply:
Sorry about the delay on this -- missed these comments somehow. Updated the PR not to inline the content